### PR TITLE
Reland Clang 11, roll buildroot to 1bc40a5f916adb93d87e2a2230a0ec43db876c1c, take 3

### PR DESCRIPTION
**Pull request by dnfield**
Changes since the last patch:

- Skip some more golden tests on mac (they didn't run last time because the C++ test failures ahead of them killed the runner).
- Add a couple missing whitelist symbols for android_jit_release_x86 
- Fix up the create_flutter_framework script to use the dsymutil from buildroot, rather than from Xcode. Xcode's dsymutil is segfaulting when mixed with the newer Clang version - it seems like you have to use the later version dsymutil.  Tested and working locally.

See also: flutter/buildroot@1bc40a5f916adb93d87e2a2230a0ec43db876c1c, #17464

### CODE REVIEW

File: testing/symbols/verify_exported.dart [hunk 1]
```diff
@@ -110,8 +110,93 @@ int _checkAndroid(String outPath, String nmPath, Iterable<String> builds) {
...
+      '__udivsi3': 'T',
+      // arm64
+      '__clz_tab': 'R',
+      '__udivti3': 'T',
+      // arm64 && x64
+      '__emutls_get_address': 'T',
+      '__emutls_register_common': 'T',
+      // jit x86
+      '__moddi3': 'T',
+      '__umoddi3': 'T',
```
Comment (dnfield) [inline testing/symbols/verify_exported.dart:191]: The new symbols to whitelist.

I manually checked x64, that's fine with the current list.

File: sky/tools/create_ios_framework.py [hunk 2]
```diff
@@ -33,27 +36,31 @@ def main():
   simulator_dylib = os.path.join(simulator_framework, 'Flutter')
 
   if not os.path.isdir(arm64_framework):
-    print 'Cannot find iOS arm64 Framework at', arm64_framework
+    print('Cannot find iOS arm64 Framework at %s' % arm64_framework)
```
Comment (chinmaygarde) [inline sky/tools/create_ios_framework.py:39]: Why the change to the prints?

Comment (dnfield) [reply sky/tools/create_ios_framework.py:39]: Move them to python3 friendly syntax.  I'm assuming at some point the bots will be on python3

**Comment by dnfield** [comment]
Going to land this again without waiting fora ll presubmits, for similar reasons as last time.  Only post submits exercise all the failures this patch was addressing.

File: testing/symbols/verify_exported.dart (hunk 1 above)
Comment (petrhosek) [reply testing/symbols/verify_exported.dart:191]: This issues has been addressed in upstream by https://reviews.llvm.org/D77280
